torch/_dynamo/symbolic_convert.py

```diff
@@ -1048,6 +1048,72 @@ def _missing(opname: str, *args: Any) -> None:
         cls.dispatch_table = [dispatch_table.get(i) for i in range(2**8)]
 
 
+class Stack:
```

Comment:
This is really faster than collections.Deque?

Reply:
Also probably should subclass some abstract classes like Sized, MutableSequence, etc